fix(android): added recent apps button

NareshMurthy opened the pull request:
Added new android keys for playwright-android - AppSwitch, Assist, Cut, Copy, Paste
Modified android key from 'At' to '@'
Fixes #5216

NareshMurthy committed:
fix(android): added recent apps button
Added a new android key - RecentApps
Fixes #5216

pavelfeldman commented:
They call it [KEYCODE_APP_SWITCH](https://developer.android.com/reference/android/view/KeyEvent#KEYCODE_APP_SWITCH), should we rename to AppSwitch?

Also, while you are there, could you add these:
...
['@', 77],
...
['AppSwitch', 187],
['Assist', 219],
['Cut', 277],
['Copy', 278],
['Paste', 279],

I had them in a patch that never got uploaded.

NareshMurthy committed:
fix(android): added buttons
Added new android keys for playwright-android - AppSwitch, Assist, Cut, Copy, Paste
Modified a android key from 'At' to '@'
Fixes #5216

NareshMurthy commented:
@pavelfeldman I've added the below keys

New keys
['AppSwitch', 187],
['Assist', 219],
['Cut', 277],
['Copy', 278],
['Paste', 279],

Modified 
['At', 77], to ['@', 77],

pavelfeldman commented:
Oh, missed it, let me try resolving the conflict and landing it!
